PDBminer

```diff
@@ -89,6 +89,13 @@ parser.add_argument("-p", '--peptide_length',
                     default = 5,
                     help='The minimum length of a peptide to be aligned. It is recomended not to go below 5 due to the risk of false alignment of a very small peptide')
 
+parser.add_argument("-a", "--file_save",
+                    metavar = "File save strategy",
```

Comment:
[typically `metavar`s are upper case and a single word](https://docs.python.org/3/library/argparse.html#metavar). This is becaue they are displayed in help as possible arguments for the option. For instance:

```
  -m MUTLIST, --mutlist MUTLIST
                        file containing the residue types that each residue
                        will be mutated to
```

here `MUTLIST` is our `metavar`